lib/assertions/is-not-required.js

```diff
@@ -0,0 +1,16 @@
+import elementToString from '../helpers/element-to-string';
+
+export default function notRequired(message) {
+  let element = this.findTargetElement();
+  if (!element) return;
+
+  let result = element.required === false;
```

Comment:
what happens if the element is not an `<input>` element? would `required` be `undefined` then and fail this check?

Reply:
Good catch. I've added a conditional, which will throw if an element is passed that is not one of `HTMLInputElement`, `HTMLTextAreaElement`, or `HTMLSelectElement`.

Reply:
And a test...